.vscode/settings.json

```diff
@@ -6,5 +6,5 @@
     "source.formatDocument": "explicit"
   },
   "prettier.useEditorConfig": true,
-  "files.insertFinalNewline": false
```

Comment:
Title says `disable insertion of final newline` but this seems to be the opposite...?

Either way's fine, though. Since I think I may be the only one not actively using vscode on this repo:

I'll leave it to someone who does to sign off on if `formatOnSave` should be introduced as a repo-level config or if it's better to just keep it at the removal and leave that to user config.